openmp/scripts/get_gcc_ver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+. ../config.mk;
```

Comment:
This is, strictly speaking, not legal.  The quoting of shell scripts and make is different.  If you need the value of CC, you need to use `make` to print it.